lib/UrlQueryHelper.js

```diff
@@ -93,6 +93,18 @@ export default class UrlQueryHelper {
     stringArray: str => {
       return !str ? null : str.split(',');
     },
+    /**
+     * @param {string} str
+     * @returns {number}
+     */
+    float: str => {
+      if (isNil(str) || !str.length) {
+        return null;
+      } else {
+        const result = parseFloat(str);
+        return isNaN(result) || result;
```

Comment:
Let's not return a boolean, which would equal to `1.0`

```suggestion
        return isNaN(result) ? null : result;
```

Reply:
Done. 👍🏽